docs/using/security-practices.md

```diff
@@ -141,17 +141,9 @@ channel by leaking plain bits -- in technical terms, _capabilities_ in
 Sandstorm are never just bits, and therefore you cannot leak
 _capabilities_ via covert channels.
 
-_**Beta Notice:** As of this writing (April 2016), Sandstorm is in
-beta. Key features allowing a user to easily grant an application
-access to external resources are still in development. In order to
-make Sandstorm more useful to early adopters, we have temporarily
-opened some intentional holes in our confinement model. For example,
-we have allowed outgoing HTTP to arbitrary servers in order to permit
-the TinyTiny RSS app to fetch RSS feeds, and we have allowed incoming
-and outgoing SMTP (with certain restrictions) to allow email clients
-to work. These holes will be closed as soon as the Powerbox UI and
-drivers make them obsolete, but in the meantime Sandstorm does not yet
-implement true confinement._
+_**Notice:** As of this writing (April 2021), Sandstorm retains limited
+incoming and outgoing SMTP to allow email clients to work. This will be
+removed once email support via the Powerbox is implemented._
```

Comment:
I do think we should still more strongly flag the fact that the network isolation is not fully implemented here as well, perhaps noting that the CSP feature still needs to be finished.

Reply:
I think my latest commit addresses this. I didn't want to go into detail twice in the document, so I referred to the latter comment. I also (think) I fixed a link which was reported broken on IRC.